external_search.py

```diff
@@ -222,7 +227,6 @@ def __init__(self, _db, url=None, works_index=None, test_search_term=None,
         if works_index and integration and not in_testing:
             try:
                 self.set_works_index_and_alias(_db)
-                self.set_stored_scripts()
```

Comment:
It doesn't make a difference, since set_works_index_and_alias is only called once, but setting the stored scripts we'll be using is part of the job of initializing the index.